engine/resources/lua/Factory/MOAIModelCreator.lua

```diff
@@ -29,28 +30,114 @@ local function objToMesh(obj)
         writeVertex(vbo, face.v3, {x = 0, y = 0}, color)
     end
 
-    -- vbo:bless()
-
     local mesh = MOAIMesh.new();
     mesh:setVertexBuffer(vbo, vertexFormat);
     mesh:setPrimType(MOAIMesh.GL_TRIANGLES);
-
+    mesh:setTotalElements ( vbo:countElements ( vertexFormat ))
+    mesh:setBounds ( vbo:computeBounds ( vertexFormat ))
     return mesh;
+    -- return makeCube(128);
+end
+
+function makeBoxMesh ( xMin, yMin, zMin, xMax, yMax, zMax, texture )

```

Comment:
If this was for testing purposes remove it. It is easy enough to find in the samples in the future.